tutoraspects/templates/aspects/apps/superset/docker/docker-bootstrap.sh

```diff
@@ -47,7 +47,7 @@ elif [[ "${1}" == "beat" ]]; then
   celery --app=superset.tasks.celery_app:app beat --pidfile /tmp/celerybeat.pid -l INFO -s "${SUPERSET_HOME}"/celerybeat-schedule
 elif [[ "${1}" == "app" ]]; then
   echo "Starting web app..."
-  flask run -p 8088 --with-threads --reload --debugger --host=0.0.0.0
+  flask run -p 8088 --with-threads --reload --debugger --debug --host=0.0.0.0
```

Comment:
Is this meant to stay in?

Reply:
Yes, this only will affect dev environment, local en k8s works the same as before.